api_app/analyzers_manager/observable_analyzers/hunting_abuse.py

```diff
@@ -0,0 +1,58 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+
+import logging
+
+import requests
+
+from api_app.analyzers_manager.classes import ObservableAnalyzer
+from tests.mock_utils import MockUpResponse, if_mock_connections, patch
+
+logger = logging.getLogger(__name__)
+
+
+class HuntingAbuseAPI(ObservableAnalyzer):
+    url: str = "https://hunting-api.abuse.ch/api/v1/"
+    _auth_key: str
+
+    @classmethod
+    def update(cls):
+        pass
+
+    def run(self):
+        headers = {"Content-Type": "application/json", "Auth-Key": self._auth_key}
+
+        data = {"query": "get_fplist", "format": "json"}
+
+        response = requests.post(self.url, json=data, headers=headers)
+        response.raise_for_status()
+        fp_list = response.json()
+
+        for _key, value_dict in fp_list.items():
+            logger.info(f"Fetching fp_status for {self.observable_name}")
+            if value_dict["entry_value"] == self.observable_name:
+                return {"fp_status": "true", "details": value_dict}
```

Comment:
```suggestion
                return {"fp_status": True, "details": value_dict}
```
So it gets automatically converted in JSON compatible format